views/mat-row.es

```diff
@@ -103,7 +103,7 @@ const MatRow = ({ stage, day, assistants, upgrade, items, development, improveme
               <ItemIcon
                 item={item}
               />
-                {__r(item.name)}
+              {__r(item.name)}{typeof item.available === 'number' ? ' ('+ __("Available") + ":  "+item.available+')' : null}
```

Comment:
`typeof item.available === 'number' && ` (${__('Available: {{count}}', { count: item.available })}

Reply:
By using i18next we could get above code simplified